docs/guides/browser-extension/sign-in-with-ethereum.md

```diff
@@ -0,0 +1,101 @@
+---
+sidebar_label: 'Sign-in With Ethereum'
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Sign-in with Ethereum
+
+This guide will teach you how to sign a message using the [Sign-In with Ethereum](https://eips.ethereum.org/EIPS/eip-4361) standard to be able to sign human-readable messages and get their signature verified on the Universal Profile.
```

Comment:
Not sure if I'm clear what `get their signature verified on the Universal Profile.` means. 

Maybe:
```suggestion
This guide will teach you how to sign a message using the [Sign-In with Ethereum](https://eips.ethereum.org/EIPS/eip-4361) standard to be able to sign human-readable messages to verify a Universal Profile.
``` ?

Reply:
How about this proposed change @CallumGrindle @skimaharvey @Hugoo ?

```suggestion
This guide will teach you how to:
1. sign a message using the [Sign-In with Ethereum](https://eips.ethereum.org/EIPS/eip-4361) standard to be able to sign human-readable messages.
2. use a Universal Profile to verify if the signature is valid.
```